andela/bad_ass-ah-backend
# Chore(refactoring): Refactor all the codebase

**What does this PR do?**

This Pr is for refactoring our codebase

**How should this be manually tested?**

- Make sure to run the local server with the command `npm run dev`

- Test all the endpoint according to the document : [Author haven documentation](https://app.swaggerhub.com/apis/badass9/Authors-Heaven/1.0.0#/)

**What are the relevant pivotal tracker stories?**

- [ ] [165819860](https://www.pivotaltracker.com/story/show/165819860)
